eureka/S5_lightcurve_fitting/likelihood.py

```diff
@@ -272,3 +255,29 @@ def computeRMS(data, maxnbins=None, binstep=1, isrmserr=False):
         return rms, stderr, binsz, rmserr
     else:
         return rms, stderr, binsz
+
+def GP_loglikelihood(model, fit):
+    """Compute likelihood, when model fit includes GP
+     
+    Parameters
+    ----------
+    model: CompositeModel object
+        The model including the GP model
+    fit: ndarray
+        the evaluated model without the GP
+         
+    Returns
+    -------
+    likelihood of the model
+    
+    Notes
+    -----
+    History:
+    
+    - March 11, 2022 Eva-Maria Ahrer
+        moved code from Model.py
+    """
+    for model in self.components:
```

Comment:
This needs to be updated since self is no longer defined. And careful as you have two different `model` variables now